src/App.jsx

```diff
@@ -17,22 +17,80 @@ const Films = React.lazy(() => import('./components/films'));
 
 const App = () => {
   const [darkTheme, setDarkTheme] = useState(false);
+  const [films, setFilms] = useState([]);
+  const [userFilms, setUserFilms] = useState([]);
+  const [nextPage, setNextPage] = useState(1);
+
+  useEffect(() => {
+    setUserFilms(JSON.parse(localStorage.getItem('userFilms')));
+  }, []);
+
+  useEffect(() => {
+    if (!userFilms) {
```

Comment:
now you can be sure userFilms always either array with values or an empty array, so you don't need if/else

`localStorage.setItem('userFilms', JSON.stringify(userFilms));`

Reply:
done